lib/components/ArtistScreen/artist_screen_content.dart

```diff
@@ -248,143 +248,175 @@ class _ArtistScreenContentState extends State<ArtistScreenContent> {
                     ),
-                  ),
-                ),
-              ),
-            ),
-            if (_showAlbums && albums.isNotEmpty)
-            AlbumsSliverList(
-              childrenForList: albums,
-              parent: widget.parent,
-            ),
-            if (appearsOnAlbums.isNotEmpty)
-            SliverPadding(
-              padding: const EdgeInsets.fromLTRB(6, 12, 6, 0),
-              sliver: SliverToBoxAdapter(
-                child: GestureDetector(
-                  onTap: () {
-                    setState(() {
-                      _showAppearsOnAlbums = !_showAppearsOnAlbums;
-                    });
-                  },
-                  child: Padding(
-                    padding: const EdgeInsets.symmetric(horizontal: 2, vertical: 12),
-                    child: Row(
-                      crossAxisAlignment: CrossAxisAlignment.center,
-                      children: [
-                        Transform.rotate(
-                          angle: _showAppearsOnAlbums ? 0 : -math.pi / 2,
-                          child: const Icon(Icons.arrow_drop_down, size: 24),
-                        ),
-                        const SizedBox(width: 4),
-                        Text(
-                          AppLocalizations.of(context)!.appearsOnAlbums,
-                          style: const TextStyle(fontSize: 18, fontWeight: FontWeight.bold),
-                        ),
-                      ],
+                    const SizedBox(width: 4),
+                    Text(
+                      AppLocalizations.of(context)!.topTracks,
+                      style: const TextStyle(fontSize: 18, fontWeight: FontWeight.bold),
                     ),
-                  ),
+                  ],
                 ),
               ),
             ),
-            if (_showAppearsOnAlbums && appearsOnAlbums.isNotEmpty)
-            AlbumsSliverList(
-              childrenForList: appearsOnAlbums,
-              parent: widget.parent,
-            ),
-            if (albums.isEmpty && appearsOnAlbums.isEmpty)
-            SliverPadding(
-              padding: const EdgeInsets.fromLTRB(6, 12, 6, 0), // Keeping horizontal and vertical padding the same
-              sliver: SliverToBoxAdapter(
-                child: Padding(
-                  padding: const EdgeInsets.symmetric(horizontal: 2, vertical: 12), // Updated inner padding
-                  child: Center(
-                    child: Text(
-                      AppLocalizations.of(context)!.emptyFilteredListTitle,
+          ),
+        ),
+        if (_showTopTracks)
+          TracksSliverList(
+            childrenForList: widget.topTracks,
+            childrenForQueue: widget.childrenForQueue,
+            showPlayCount: true,
+            isOnArtistScreen: true,
+            parent: widget.parent,
+          ),
+      ],
+    );
+  }
+}
+
+class BuildAlbumSection extends StatefulWidget {
```

Comment:
Widgets shouldn't be called "Build...". You use "build..." when it's a function returning a widget :)